introduction_to_amazon_algorithms/xgboost_abalone/xgboost_inferenece_script_mode.ipynb

```diff
@@ -12,7 +12,9 @@
     "## Contents\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    model_data = xgb_script_mode_estimator.model_data
</pre><p>Was wondering, with which model this will create an endpoint?, why we do not have a flag "deploy_amt_model" as other notebooks have?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Good catch. I corrected that to stay consistent with the rest.